go/logic/applier_test.go

```diff
@@ -185,3 +192,181 @@ func TestApplierInstantDDL(t *testing.T) {
 		require.Equal(t, "ALTER /* gh-ost */ TABLE `test`.`mytable` ADD INDEX (foo), ALGORITHM=INSTANT", stmt)
 	})
 }
+
+type ApplierTestSuite struct {
+	suite.Suite
+
+	mysqlContainer testcontainers.Container
+}
+
+func (suite *ApplierTestSuite) SetupSuite() {
+	ctx := context.Background()
+	req := testcontainers.ContainerRequest{
```

Comment:
Hello from Testcontainers Go community! 👋  Thanks for using us 

I'm here just to suggest using the dedicated MySQL module, if possible: https://golang.testcontainers.org/modules/mysql/

And please let us know if you need anything from our side 🙏

Reply:
👋  thanks for the suggestion! We will look into using this.